Web/Angularjs-Assignment/WebContent/customer.html

```diff
@@ -0,0 +1,125 @@
+<script src="http://cdnjs.cloudflare.com/ajax/libs/angular-ui/0.4.0/angular-ui.min.js"></script>
+<script
+	src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script
+	src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+<script src="apps/app.js"></script>
+<script src="apps/controller/controller.js"></script>
+<link rel="stylesheet" href="css/custom.css">
+
+
+</head>
+<body ng-controller="customermanagement">
+	<div class="container">
+		<h1>Customers</h1>
+		<ul class="nav nav-pills">
+			<li><button href="#" ng-click="cardview()"
+					class="btn btn-default">
+					<span class="glyphicon glyphicon-th-large"></span> Card View
+				</button></li>
+			<li class="active"><button ng-click="listview()"
+					class="btn btn-default">
+					<span class="glyphicon glyphicon-list"></span> List View
+				</button></li>
+			<li><a href="#" data-toggle="modal" data-target="#myModal"><span
+					class="glyphicon glyphicon-plus"></span> Add Customers</a></li>
+			<li>
+				<div class="form-inline">
+					<label> Filter: <input type="text" class="form-control"	ng-model="searchFilter"></label>
+				</div>
+			</li>
+		</ul>
+	</div>
+	<div>
+
+		<!-- Modal -->
+		<div id="myModal" class="modal fade" role="dialog">
+			<div class="modal-dialog">
+				<!-- Modal content-->
+				<div class="modal-content">
+					<div class="modal-header">
+						<button type="button" class="close" data-dismiss="modal">&times;</button>
+						<h4 class="modal-title">Add Customers</h4>
+					</div>
+					<div class="modal-body">
+						<div class="form-group">
+							<label> First Name <input type="text"
+								ng-model="customer.fname" class="form-control" required>
+							</label>
+						</div>
+						<div class="form-group">
+							<label> Last Name <input type="text"
+								ng-model="customer.lname" class="form-control" required>
+							</label>
+						</div>
+						<div class="form-group">
+							<label> Email <input type="text"
+								ng-model="customer.email" class="form-control" required>
+							</label>
+						</div>
+						<div class="form-group">
+							<label> Address <textarea ng-model="customer.address"
+									class="form-control" required></textarea>
+							</label>
+						</div>
+						<div class="form-group">
+							<label> Number Of Orders <input type="number" min="0"
+								class="form-control" ng-model="customer.orders" required>
+							</label>
+						</div>
+						<input type="button" class="btn btn-primary"
+							ng-click="newcustomer()" Value="Add New Customer">
+
+					</div>
+
+				</div>
+			</div>
+		</div>
+
+		<div class="container customerlist">
+			<div class="panel panel-default" id="panel{{$index}}" ng-class="listview ? 'col-lg-10' : 'col-lg-4'"	ng-repeat="customer in customers | filter : filtering">
+				<div data-toggle="modal" data-target="#detailModal" ng-click="setDetails($index)">
```

Comment:
1. No Custom Filter is implemented 
2. Don't use Jquery to access Dom element use custom directive